packages/loaders/src/zarr/ome-zarr.ts

```diff
@@ -25,9 +25,16 @@ interface Omero {
   name?: string;
 }
 
+// See https://ngff.openmicroscopy.org/latest/#axes-md
+export interface Axis {
+  name: string;
+  type?: string;
+  unit?: string;
+}
+
 interface Multiscale {
   datasets: { path: string }[];
-  axes?: Labels<string[]>;
+  axes?: Labels<string[]> | Axis[];
```

Comment:
I think we should probably change the type here:

```suggestion
  axes?: string[] | Axis[];
```

We have no guarantees about the ordering or axes anymore. Internally we can still cast to the `Labels` type system, but it might be worth while to add some runtime checks that ensure this.

Reply:
Oh great point.  I guess `x` and `y` aren't even specified in the spec anymore.

Reply:
Yes so as of 0.3, the dimension order is no longer even specified so definitely fine with this.